syntax = "proto3";

package auth;

service AuthService {
  rpc RegisterUser(RegisterRequest) returns (RegisterResponse) {}
  rpc LoginUser(LoginRequest) returns (LoginResponse) {}
}

message RegisterRequest {
  string email = 1;
  string password = 2;
}

message RegisterResponse {
  string message = 1;
}

message LoginRequest {
  string email = 1;
  string password = 2;
}

message LoginResponse {
  string message = 1;
  string access_token = 2;
}